src/app/work-item/work-item.service.spec.ts

```diff
@@ -95,6 +96,24 @@ describe('Work Item Service - ', () => {
       });
   }));
 
+
+  it('Get work item types', async(() => {
+    mockService.connections.subscribe((connection: any) => {
+      connection.mockRespond(new Response(
+        new ResponseOptions({
+          body: JSON.stringify(response),
```

Comment:
The response defined up there is for work items, not work item types. You need to define a mock response for the types. The test will pass because it's being tested against the same data but this is not how it should be done.